step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageFileSource.java

```diff
@@ -56,6 +65,12 @@ public static AutomationPackageFileSource withResourceId(String resourceId){
         return res;
     }
 
+    public static AutomationPackageFileSource withManagedLibraryName(String managedLibraryName){
```

Comment:
"with..." is usually used for builder patterns and setting individual fields. Suggest to rename to "forManagedLibraryName" (also applies to other similarly named methods).